lib/charts/besu-prysm/templates/besu.statefulset.yaml

```diff
@@ -60,7 +52,6 @@ spec:
             - "--data-storage-format=BONSAI"
             - "--rpc-tx-feecap=0"
             - "--logging={{ .Values.excutionClient.logging.level }}"
-            - "--kzg-trusted-setup=/data/custom_config_data/trusted_setup.txt"
```

Comment:
having this flag was causing an error: 

```
2024-09-13 15:05:32.947+00:00 | main | ERROR | Besu | Failed to start Besu
java.lang.RuntimeException: Couldn't load Trusted Setup. File might not exist or there is a permission issue.
	at ethereum.ckzg4844.CKZG4844JNI.loadTrustedSetup(Native Method)
	at org.hyperledger.besu.evm.precompile.KZGPointEvalPrecompiledContract.init(KZGPointEvalPrecompiledContract.java:63)
	at org.hyperledger.besu.cli.BesuCommand.configureNativeLibs(BesuCommand.java:1542)
	at org.hyperledger.besu.cli.BesuCommand.run(BesuCommand.java:1193)
	at picocli.CommandLine.executeUserObject(CommandLine.java:2026)
	at picocli.CommandLine.access$1500(CommandLine.java:148)
```
Not sure if that was related to my changes or the upgrade of besu to the newer version.
After removing the flag it started building blocks fine, and everything worked.

@Tofel Do we need that?

Reply:
path was incorrect, it should be `"--kzg-trusted-setup=/data/metadata/trusted_setup.txt"`

this param is required only by some Besu versions.